column_buffer.go

```diff
@@ -1999,16 +1998,16 @@ type writeRowsFunc func(columns []ColumnBuffer, rows sparse.Array, levels column
 // writeRowsFuncOf generates a writeRowsFunc function for the given Go type and
 // parquet schema. The column path indicates the column that the function is
 // being generated for in the parquet schema.
-func writeRowsFuncOf(t reflect.Type, schema *Schema, path columnPath) writeRowsFunc {
+func writeRowsFuncOf(t reflect.Type, schema *Schema, path columnPath, tagReplacements []StructTagOption) writeRowsFunc {
```

Comment:
`tagReplacements []StructTagOption`

We should rename one of these so they are consistent. Any preference?